src/components/sidebar/SidebarNavigation/index.tsx

```diff
@@ -31,6 +32,19 @@ const isRouteEnabled = (route: string, chain?: ChainInfo) => {
   return !featureRoute || hasFeature(chain, featureRoute)
 }
 
+const NavItemLabel = ({ item }: { item: NavItem }) => {
+  const queueSize = useQueuedTxsLength()
+
+  if (item.href === AppRoutes.transactions.history) {
+    return <SidebarListItemCounter count={queueSize} />
+  }
+
+  if (item.href === AppRoutes.swap) {
+    return <Chip label="New" color="success" sx={{ ml: 1 }} />
```

Comment:
@TanyaEfremova what chip style should be used here? We've used this one in other places:
<img width="260" alt="Screenshot 2024-05-29 at 10 38 37" src="https://github.com/safe-global/safe-wallet-web/assets/381895/2eb6cfa5-150a-42a8-a08b-481bc52c8add">

Reply:
Yes, I initially went for the "new chip" that we have, but that is not what is in the figma design.